src/components/molecules/TodoActions.vue

```diff
@@ -7,39 +7,74 @@ const store = useStore()
 
 const filters = computed(() => store.getters.getFilters)
 const remaining = computed(() => store.getters.undoneTodosCount)
+const visibility = computed(() => store.getters.getVisibility)
 const clearCompleted = () => store.dispatch('clearCompleted')
 const changeVisibility = (visibility) => store.dispatch('changeVisibility', visibility)
 
 const capitalize = (s) => s.charAt(0).toUpperCase() + s.slice(1)
+
+const remainingText = (remaining) => {
+  switch (remaining) {
+    case 0:
+      return 'No items left'
+    case 1:
+      return '1 item left'
+    default:
+      return remaining + ' items left'
+  }
+}
 </script>
 
 <template>
-  <div class="task-actions">
-    <span>{{ remaining }} items left</span>
-    <ul>
-      <AppButton
-        v-for="(val, key) in filters"
-        :key="key"
-        :label="capitalize(key)"
-        @click="changeVisibility(key)"
-        class="b"
-      />
-    </ul>
-    <AppButton label="Clear Completed" @click="clearCompleted" />
+  <div class="card-body">
+    <div class="d-flex justify-content-between align-items-center">
+      <span class="remaining">{{ remainingText(remaining) }}</span>
+      <div>
+        <AppButton
+          v-for="(val, key) in filters"
+          :key="key"
+          :customClass="[
+            'btn-sm',
+            'me-2',
+            {
+              active: visibility === key,
```

Comment:
Please, create, some kind of enum (e.g. Object.freeze()) for filter names ('all', 'active', 'completed'). You can store it in constants.js file.
It might look something like this
Object.freeze({
  ALL: 'all',
  ACTIVE: 'active',
  COMPLETED: 'completed',
})